packages/fluentui/docs/src/examples/components/SplitButton/index.tsx

```diff
@@ -3,9 +3,11 @@ import * as React from 'react';
 import Types from './Types';
 import States from './States';
 import Slots from './Slots';
+import Usage from './Usage';
 
 const SplitButtonExamples = () => (
   <>
+    <Usage />
```

Comment:
This section should be last (after `slots`)

Reply:
Done